pom.xml

```diff
@@ -67,7 +67,7 @@
         <copyrightNotice>Copyright ${project.inceptionYear}-${copyrightYear} Micro Focus or one of its affiliates.</copyrightNotice>
 
         <!--BOM dependencies versions-->
-        <caf.worker-document-framework.bom.version>4.6.0-SNAPSHOT</caf.worker-document-framework.bom.version>
+        <caf.worker-document-framework.bom.version>5.0.0-SNAPSHOT</caf.worker-document-framework.bom.version>
```

Comment:
@dermot-hardy I'm not sure if a breaking change to a dependency here should mean a breaking change in this project (caf-dependency-management-bom) as well? Should I update this project to 3.0.0-SNAPSHOT?

Reply:
I agree that it is.